connection.js

```diff
@@ -416,20 +415,13 @@ class Connection {
         this._process_data();
     }
     _process_data () {
-        const self = this;
         // We *must* detect disconnected connections here as the state
         // only transitions to states.CMD in the respond function below.
         // Otherwise if multiple commands are pipelined and then the
         // connection is dropped; we'll end up in the function forever.
         if (this.state >= states.DISCONNECTING) return;
 
-        let maxlength;
-        if (this.state === states.PAUSE_DATA || this.state === states.DATA) {
-            maxlength = this.max_data_line_length;
-        }
-        else {
-            maxlength = this.max_line_length;
-        }
+        const maxlength = this.state === states.PAUSE_DATA || this.state === states.DATA ? this.max_data_line_length : this.max_line_length;
 
```

Comment:
That code is nice and succinct but it's not very easy to reason about, and brutal to review on a mobile device. How about this instead?
```suggestion
        let maxlength = this.max_line_length;
        if (this.state === states.PAUSE_DATA || this.state === states.DATA) {
            maxlength = this.max_data_line_length;
        }
```